### PR TITLE
feat: add story divider

### PR DESCRIPTION
**Pull request by vinzkm**
-add story for divider component
-include a theme for divider

### CODE REVIEW

File: src/theme/components/Divider.ts
Comment (rnd500designs) [inline src/theme/components/Divider.ts:1]: for any color, refer to pallete.ts, or the standard way of theming the colors in MUI.
all hex or rgb values must be stored in reusable variables. :)

File: src/components/Divider.stories.tsx
Comment (rnd500designs) [inline src/components/Divider.stories.tsx:1]: in figma there's a subheader variant. you may include this in the variants and style per the mockup. 
part of the usage in diveder is on subheaders or section per the design system

**Review by rnd500designs — CHANGES_REQUESTED** [review]

File: src/components/Divider.stories.tsx
Comment (vinzkm) [reply src/components/Divider.stories.tsx:1]: just a clarification, can it be as a story (With Subheader story) instead? Then return a divider and a typography below it for that specific story. 
I was checking the divider's API if it includes a sub-header but the closest is the 'children' prop but it adds the text in the divider itself.

Comment (rnd500designs) [reply src/components/Divider.stories.tsx:1]: just rename the story from "with subheader" to "Subheader " to match figma.
it means that the divider is for a subheader not necessarily having a sub header.
it seems to match the "inset" variant of the mui divider component.

**Review by rnd500designs — CHANGES_REQUESTED** [review]

**Review by rnd500designs — APPROVED** [review]

**Pull request merged**